xrtpy/response/channel.py

```diff
@@ -73,203 +76,165 @@ def __init__(self, index):
 
     @property
     def entrancefilter_name(self,) -> str:
-        """
-        Entrance filter name.
-        """
+        """Entrance filter name."""
         return self._en_filter_data["LONG_NAME"]
 
     @property
     def entrancefilter_material(self) -> str:
-        """
-        XRT channel filter material.
-        """
+        """XRT entrance filter material."""
         return self._en_filter_data['MATERIAL']
 
     @property 
     @u.quantity_input
     def entrancefilter_thickness(self) -> u.angstrom:
-        """
-        XRT channel filter material thickness.
-        """
+        """XRT entrance filter material thickness."""
         return u.Quantity(self._en_filter_data['THICK'],u.angstrom)
     
     @property
     def entrancefilter_density(self)-> u.g * u.cm**-3 :
-        """
-        XRT channel filter material density. g cm^-3
-        """
+        """XRT entrance filter material density. g cm^-3"""
```

Comment:
```suggestion
        """XRT entrance filter material density in g cm\ :math:`^{-3}`."""
```
Assuming I'm remembering reST formatting right...